srcs/termcaps/termcaps_read_input.c

```diff
@@ -108,6 +108,7 @@ static int		s_termcaps_read_loop(const int fd)
 
 	while (g_context.state != STATE_EXIT && g_context.state != STATE_CONTINUE)
 	{
+		job_background_update_status();
 		input_buffer_size = read(fd, input_buffer, 1);
 		ASSERT(input_buffer_size == 1);
 		s_termcaps_identify_input(input_buffer[0],
```

Comment:
@abombard @gabkk 
We will have something like this to put at this place.
And surely we would like to have this loop to not wait for 1 character by using `VMIN == 0` :-)
Or find another way (write something that could be catch by your read)
see you ..